src/test/java/com/dream11/rest/RestApiIT.java

```diff
@@ -0,0 +1,103 @@
+package com.dream11.rest;
+
+import com.dream11.rest.Constants;
+import com.dream11.rest.Setup;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.VertxExtension;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
```

Comment:
Are these tests working with vertx webclient?

Reply:
Have not tried using vertx webclient